### DISCUSSION TITLE
paginationView doesn't work

### DISCUSSION THREAD

**Discussion by vincentdev15** [livewire/livewire#5851] [7. Bugs (archived)]
Even if I add the paginationView function and return a custom pagination view, it doesn't work.

But it works via the `->links('custom-view')` helper.

**Comment by joshhanley** [comment]
@vincentdev15 thanks for reporting! Any chance you could submit a PR with a failing test? Thanks!

**Reply by vincentdev15** [reply]
Sorry I can't, I just noticed that the custom view isn't taken into account.
